packages/main/src/components/FlexBox/FlexBox.cy.tsx

```diff
@@ -93,6 +93,48 @@ describe('FlexBox', () => {
       .should('have.css', 'height', '200px');
   });
 
+  it('gap values', () => {
+    cy.mount(
+      <FlexBox data-testid="flexbox">
+        <span>Test 1</span>
+        <span>Test 2</span>
+        <span>Test 3</span>
+      </FlexBox>
+    );
+
+    cy.findByTestId('flexbox').should('have.css', 'gap', 'normal');
```

Comment:
**Note:** [`normal`](https://developer.mozilla.org/en-US/docs/Web/CSS/gap) should be the default value according to various sources. Therefore, this shall not cause a breaking change for existing components.